### PR TITLE
chore: Add order to content item connections

### PR DESCRIPTION
**Pull request by tomthornton**

### CODE REVIEW

File: dags/rock_content_items_connections.py [hunk 1]
```diff
@@ -86,13 +86,14 @@ def update_content(obj):
                 'ContentItemsConnection',
                 get_postgres_id(obj['ChildContentChannelItemId']),
                 get_postgres_id(obj['ContentChannelItemId']),
+                obj['Order']
```
Comment (vinnyjth) [inline dags/rock_content_items_connections.py:89]: Can you add a reference to this field to the comment on line 79? I've been trying to do that to make code easier to parse.

File: dags/rock_content_items_connections.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def get_postgres_id(id):
             "$top": top,
             "$skip": skip,
             # "$expand": "Photo",
-            "$select": "Id,ChildContentChannelItemId,ContentChannelItemId",
+            "$select": "Id,ChildContentChannelItemId,ContentChannelItemId,Order",
```
Comment (vinnyjth) [inline dags/rock_content_items_connections.py:52]: 👍

**Review by vinnyjth — CHANGES_REQUESTED** [review]
Looking good! One teeny tiny tweak :)

File: dags/rock_content_items_connections.py (hunk 1 above)
Comment (tomthornton) [reply dags/rock_content_items_connections.py:89]: @vinnyjth yep!

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**